browser.js

```diff
@@ -1,5 +1,24 @@
+/* eslint-env browser */
 'use strict';
+
+function getChromeVersion() {
+	const matches = /Chrom(e|ium)\/(?<chromeVersion>\d+)\./.exec(navigator && navigator.userAgent);
```

Comment:
```suggestion
	const matches = /(Chrome|Chromium)\/(?<chromeVersion>\d+)\./.exec(navigator && navigator.userAgent);
```

Reply:
In what case would `navigator` not exist in the browser?

Reply:
Depends on if we want to support IE 3 or below 🙃 

Pretty certain we don't, though.

Reply:
This addition is breaking the use over any browser that does not support ES2018.

I don't see any strong reason to use RegExp's capture groups here, where a simple split+pop would have been enough?